docs/HowTo/Get-Started/Connect-To-Testnet.md

```diff
@@ -0,0 +1,184 @@
+
+If you already have created a custom testnet or plan to do so, Teku allows you to pass different configurations to the beacon-chain node. A custom network requires a couple of things:
+
+* A network configuration that can be either one of the `minimal` or `mainnet` presets, or a custom `yaml` file containing the network specification.
+* An `ssz` genesis state or information about the ETH1 deposit contract.
+* Eventually bootnodes to allow other users to connect to your network.
+
+**Network configuration**:
+
+The network configuration can be either of the presets `minimal` and `mainnet` or a `yaml` file. The custome configuration file can be passed as local file or remote config fetched via HTTP, e.g.:
+
+* `--network="minimal"`
+* `--network="mainnet"`
+* `--network="./path/to/chain.yaml"`
+* `--network="https://github.com/goerli/schlesi/raw/master/teku/chain.yaml"`
+
+!!! example
+
+    ```
+    teku --network="https://github.com/goerli/schlesi/raw/master/teku/chain.yaml"
+    ```
+
+**Deposit Contract**:
+
+The deposit contract is a deployed instance of the [`validator_registration.vy`](https://github.com/ethereum/eth2.0-specs/blob/dev/deposit_contract/contracts/validator_registration.vy) on an ETH1 network of your choice.
+
+`--eth1-deposit-contract-address="0xaa888248144bc5d584a7f400839d0d912f21c39a"`
+
+!!! tip
+
+    See also: [How to run your own Beacon Chain](https://dev.to/q9/how-to-run-your-own-beacon-chain-e70)
+
+Note, that reading the deposit contract requires an enabled ETH1 connection, e.g.:
+
+`--eth1-enabled --eth1-endpoint="localhost:8545"`
+
+!!! example
+
+    ```
+    teku --network="minimal" --eth1-deposit-contract-address="0xaa888248144bc5d584a7f400839d0d912f21c39a" --eth1-enabled --eth1-endpoint="localhost:8545"
+    ```
+
+**Genesis State**:
+
+In case the genesis state is already known and available, instead of passing an deposit contract, the initial state can be provided:
+
+* `--initial-state="./path/to/genesis.ssz"`
+* `--initial-state="https://github.com/goerli/schlesi/raw/master/teku/genesis.ssz"`
+
+No ETH1 connection is required in case the genesis state is available.
+
+!!! example
+
+    ```
+    teku --network="minimal" --initial-state="https://github.com/goerli/schlesi/raw/master/teku/genesis.ssz"`
+    ```
+
+**Bootstrap Nodes**:
+
+To allow other developers and users to connect to your custom testnet, you can provide bootstrap nodes and static peers.
+
+* `--p2p-discovery-bootnodes="enr:-LK4QFO0gKFieMiNrUystSk5Xt7DmIgusloLudv-gH8Krjw9SsUDZRk---H-3hwvL9rMfsMcZwU6L5ezK2d1_dG0UgECh2F0dG5ldHOIAAAAAAAAAACEZXRoMpCZJe_WAAAAAP__________gmlkgnY0gmlwhDMPd52Jc2VjcDI1NmsxoQPNb3TG-iN0aGTagN4peO0SEkWKklJOvloWL0He8pnB_4N0Y3CCJRyDdWRwgiUc"`
+* `--p2p-static-peers="/ip4/51.15.119.157/tcp/9000/p2p/16Uiu2HAkvLCWwVEfF365ZWXB6siDL1mUpcd1XQ1nSXAHmvM5W7wn"`
+
+If the number of command-line arguments becomes confusing, it's recommended to [Use a Configuration File](HowTo/Configure/Use-Configuration-File.md).
```

Comment:
lowercase "Use a Configuration File"

Reply:
Also the link is wrong (see CI linkchecker report on this PR). It should be `/HowTo/Configure/Use-Configuration-File.md`, starting with a `/` as otherwise it's relative to the current page position and then points to `/HowTo/Get-Started/HowTo/Configure/Use-Configuration-File.md` which doesn't exist.